src/psyclone/parse/module_manager.py

```diff
@@ -53,6 +62,10 @@ class ModuleManager:
     # Class variable to store the singleton instance
     _instance = None
 
+    # How similar a file name must be to a module name to trigger reading
+    # of the file.
+    _threshold_similarity = 0.7
```

Comment:
That is a really great idea. I can't wait to see how this works with LFRic and UM :)

Reply:
Fingers crossed! At least the integration tests are happy.